daml-script/dump/src/test/scala/com/daml/script/dump/EncodeTreeSpec.scala

```diff
@@ -46,10 +47,127 @@ class EncodeTreeSpec extends AnyFreeSpec with Matchers {
           traceContext = None,
         )
         encodeTree(parties, cidMap, cidRefs, tree).render(80) shouldBe
-          """tree <- submitTreeMulti [alice_0] [] do
+          """submitMulti [alice_0] [] do
             |  createCmd Module.Template""".stripMargin.replace("\r\n", "\n")
       }
-      "referenced" in {
+      "unreferenced creates" in {
+        val parties = Map(Party("Alice") -> "alice_0")
+        val cidMap = Map(
+          ContractId("cid1") -> "contract_1_0",
+          ContractId("cid2") -> "contract_1_1",
+        )
+        val cidRefs = Set.empty[ContractId]
+        val tree = TransactionTree(
+          transactionId = "txid",
+          commandId = "cmdid",
+          workflowId = "flowid",
+          effectiveAt = None,
+          offset = "",
+          eventsById = Map(
+            "create1" -> TreeEvent(
+              TreeEvent.Kind.Created(
+                CreatedEvent(
+                  eventId = "create",
+                  templateId = Some(Identifier("package", "Module", "Template")),
+                  contractId = "cid1",
+                  signatories = Seq("Alice"),
+                  createArguments = Some(
+                    Record(
+                      recordId = Some(Identifier("package", "Module", "Template")),
+                      fields = Seq.empty,
+                    )
+                  ),
+                )
+              )
+            ),
```

Comment:
looks like a helper similar to `mkCreated` might come in handy. and probably also a helper to create the wrapping trees.

Reply:
I've created simple `mkCreated` and `mkExercised` helpers. We can think of something more sophisticated going forward.